qchem/pennylane_qchem/qchem/obs.py

```diff
@@ -311,65 +311,56 @@ def observable(me_table, init_term=0, mapping="jordan_wigner"):
 
     .. math::
 
         \hat{S}_z = \sum_{\alpha, \beta} \langle \alpha \vert \hat{s}_z \vert \beta \rangle
-        ~ \hat{c}_\alpha^\dagger\hat{c}_\beta,
+        ~ \hat{c}_\alpha^\dagger \hat{c}_\beta,
 
         \langle \alpha \vert \hat{s}_z \vert \beta \rangle = m_\alpha \delta_{\alpha,\beta},
 
-    where :math:`m_\alpha` refers to the quantum number of the spin wave function
-    :math:`\chi_{m_\alpha}(s_z)` of the spin-orbital :math:`\vert \alpha \rangle`
-    and :math:`\hat{c}_\alpha^\dagger` (:math:`\hat{c}_\alpha`) is the creation (annihilation)
-    particle operator acting on the :math:`\alpha`-th active orbital.
+    where :math:`m_\alpha = \pm 1/2` denotes the spin-projection quantum number of
+    the single-particle state :math:`\vert \alpha \rangle`. The operators :math:`\hat{c}^\dagger`
+    and :math:`\hat{c}` are the particle creation and annihilation operators, respectively.
 
     Args:
-        mol_name (str): name of the molecule
-        hf_data (str): path to the directory with the HF electronic structure data
-        n_active_electrons (int): number of active electrons
-        n_active_orbitals (int): number of active orbitals
+        n_orbitals (str): Number of orbitals. If an active space is defined, 'n_orbitals'
+            is the number of active orbitals.
+        mapping (str): Specifies the transformation to map the fermionic operator to the
+            Pauli basis. Input values can be ``'jordan_wigner'`` or ``'bravyi_kitaev'``.
 
     Returns:
-        array: NumPy array with the table of matrix elements. Since :math:`\hat{S}_z` is
-        diagonal in the basis of HF orbitals the first two columns contains the index
-        :math:`\alpha` and the third column stores the matrix element.
+        pennylane.Hamiltonian: the fermionic-to-qubit transformed :math:`\hat{S}_z` observable
 
     **Example**
 
-    >>> get_spinZ_matrix_elements('h2', './pyscf/sto-3g', n_active_electrons=2, n_active_orbitals=2)
-    [[ 0.   0.   0.5]
-    [ 1.   1.  -0.5]
-    [ 2.   2.   0.5]
-    [ 3.   3.  -0.5]]
+    >>> n_orbitals = 2
+    >>> Sz = spin_z(n_orbitals, mapping="jordan_wigner")
+    >>> print(Sz)
+    (-0.25) [Z0]
+    + (0.25) [Z1]
+    + (-0.25) [Z2]
+    + (0.25) [Z3]
     """
 
-    active_indices = structure.active_space(
-        mol_name,
-        hf_data,
-        n_active_electrons=n_active_electrons,
-        n_active_orbitals=n_active_orbitals,
-    )[1]
-
-    n_spin_orbs = 2 * len(active_indices)
-    sz = np.where(np.arange(n_spin_orbs) % 2 == 0, 0.5, -0.5)
+    n_spin_orbs = 2 * n_orbitals
+    sz_orb = np.where(np.arange(n_spin_orbs) % 2 == 0, 0.5, -0.5)
 
-    spinz_matrix_elements = np.zeros((n_spin_orbs, 3))
+    # build table with matrix elements
+    table = np.zeros((n_spin_orbs, 3))
     for alpha in range(n_spin_orbs):
-        spinz_matrix_elements[alpha] = np.array([alpha, alpha, sz[alpha]])
+        table[alpha] = np.array([alpha, alpha, sz_orb[alpha]])
```

Comment:
@agran2018 we can approach this calculation the same way we do in https://github.com/XanaduAI/pennylane/pull/698/files#r456225583.

In this particular case, the table will always have three columns --- the first two columns will simply be `range(n_spin_orbs)`, and the third column will always be `sz_orb`.

So we can construct this table directly, by simply stacking each column. For example,

```pycon
>>> n_spin_orbs = 8
>>> r = np.arange(n_spin_orbs)
>>> table = np.vstack([r, r, sz_orb]).T
>>> print(table)
[[ 0.   0.   0.5]
 [ 1.   1.  -0.5]
 [ 2.   2.   0.5]
 [ 3.   3.  -0.5]
 [ 4.   4.   0.5]
 [ 5.   5.  -0.5]
 [ 6.   6.   0.5]
 [ 7.   7.  -0.5]]
```

Reply:
Thanks @josh146 , these simple examples I already got them. I am ashamed ;-)